patches/fortmatic+2.2.1.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/node_modules/fortmatic/dist/cjs/fortmatic.js b/node_modules/fortmatic/dist/cjs/fortmatic.js
```

Comment:
Seem to work well. What did you change on this file exactly? It's hard to review now cuz it's all minified

Reply:
1. oh! this is because the bug cause by the fortmatic's source. 
<img width="804" alt="Screen Shot 2021-08-16 at 22 31 28" src="https://user-images.githubusercontent.com/87322530/129580507-af0402c2-bc60-49c6-a5aa-a9133e46c5c7.png">
2. then I find out the solution: https://stackoverflow.com/questions/42376464/uncaught-domexception-failed-to-execute-postmessage-on-window-an-object-co

3. so the bug come from the fortmatic's source code and I see there are some patches that made by `patch-package`

4. then I make one and I just add the `JSON.parse(JSON.stringify(` code to fix the bug